bionic/deriver.py

```diff
@@ -26,8 +26,9 @@ class EntityDeriver(object):
 
     # --- Public API.
 
-    def __init__(self, flow_state):
+    def __init__(self, flow_state, flow_uuid):
         self._flow_state = flow_state
+        self._flow_uuid = flow_uuid
```

Comment:
nit: if a `flow_uuid` is not part of a `flow_state`, maybe it should be called a `flow_instance_uuid`?